src/index.js

```diff
@@ -26,7 +26,6 @@ export var atom = derivable.atom;
 export var atomic = derivable.atomic;
 export var atomically = derivable.atomically;
 export var proxy = derivable.proxy;
-export var derive = derivable.derive;
```

Comment:
Is this a mistake?

Reply:
Nope. It was duplicated. And rollup is failed on it.

Reply:
Here's duplication
https://github.com/ds300/derivablejs/blob/master/src/index.js#L24-L29